.github/workflows/ca-sequential-test.yml

```diff
@@ -1836,6 +1836,595 @@ jobs:
 
           diff expected actual
 
+      ####################################################################################################
+      # Switch cert request ID generator to newLegacy and verify if serials
+      # have gaps when range is updated
+      #
+      # It should work like the legacy but with correct range.
+      - name: Switch to newLegacy
+        run: |
+          # Update CS.cfg
+          docker exec ds ldapsearch \
+              -H ldap://ds.example.com:3389 \
+              -D "cn=Directory Manager" \
+              -w Secret.123 \
+              -b ou=certificateRepository,ou=ranges,dc=ca,dc=pki,dc=example,dc=com \
+              -s one \
+              -o ldif_wrap=no \
+              -LLL beginRange | tee output
```

Comment:
Thanks for the update. Yes, we probably need to do something like this, but we need to make sure that the range we pick is really assigned to this server, so we need to check the hostname & port number.

I'm still trying to analyze & document the issue here:
https://github.com/dogtagpki/pki/wiki/Sequential-Serial-Numbers-v1

I'm thinking we need to create an SSNv2 page with the proper range progression, then we fix the code to work that way, then design a migration path into SSNv2 that will match the proper range progression (so that it will be indistinguishable from a server that was installed with SSNv2 from the beginning), and it should not conflict with other servers still running with SSNv1.

Reply:
> Thanks for the update. Yes, we probably need to do something like this, but we need to make sure that the range we pick is really assigned to this server, so we need to check the hostname & port number
I did not check for the hostname/port because this test was for the standalone case. For the clone case it is assigned the current and next range so more logic is required to set the new_range properly.

Reply:
Using the v1 page as model I created the page: 
https://github.com/dogtagpki/pki/wiki/Sequential-Serial-Numbers-v2